test/WebSites/WebApi/EndPoints/OpenApiEndpoints.cs

```diff
@@ -61,4 +67,6 @@ record WeatherForecast(DateOnly Date, int TemperatureC, string? Summary)
     record class Person(string FirstName, string LastName);
 
     record class Address(string Street, string City, string State, string ZipCode);
+    sealed record OrganizationCustomExchangeRatesDto([property: JsonRequired] CurrenciesRate[] CurrenciesRates);
+    sealed record CurrenciesRate([property: JsonRequired] string currencyFrom, [property: JsonRequired] string currencyTo, [property: JsonRequired] double rate);
```

Comment:
Could you make one of them _not_ required so we can see the different behaviour for the same model in the rendered contract please?